src/Microsoft.DotNet.Interactive.Document.Tests/NotebookDocumentFileFormatTestsBase.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.DotNet.Interactive.Document.Tests
+{
+    public abstract class NotebookDocumentFileFormatTestsBase
```

Comment:
This naming pattern seems more verbose than necessary. In this context, "notebook" and "document" and "file" all mean more or less the same thing.

Reply:
makes sense